docs/layout.md

```diff
@@ -34,866 +34,148 @@ Along with a fifth for:
-                - [ ] Audit log
-                - [ ] Feedback
-    - [ ] FedRAMP for SSH & K8s
-        - [ ] Setup
-        - [ ] Configuration
-        - [ ] What else does the Teleport FIPS binary enforce?
-    - [ ] Role-Based Access Control
-        - Links to Access Controls Introduction
-- [ ] Cloud Section
-    - [ ] Introduction
-        - [ ] Next steps
-        - w/ Demo video
-    - [ ] Getting Started
-        - [ ] Step 1/3. Signup
-        - [ ] Step 2/3. Install client tools
-        - [ ] Step 3/3. Explore your cluster
-        - [ ] Next steps
-    - [ ] Architecture
-        - [ ] Security
-        - [ ] Compliance
-        - [ ] Managed Teleport settings
-        - [ ] Data retention
-        - [ ] High Availability
-    - [ ] Teleport Cloud FAQ
-        - Section-specific FAQ
-- [ ] Architecture Section
-    - [ ] Architecture Overview
-        - [ ] What makes Teleport different
-        - [ ] Design principles
-        - [ ] Definitions
-        - [ ] Teleport services
-        - [ ] Basic architecture overview
-        - [ ] Detailed architecture overview
-        - [ ] Teleport CLI tools
-        - [ ] Next steps
-    - [ ] Teleport Users
-        - [ ] Types of users
-        - [ ] User roles
-        - [ ] More concepts
-    - [ ] Teleport Nodes
-        - [ ] The Node service
-        - [ ] Node identity on a cluster
-        - [ ] Connecting to Nodes
-        - [ ] Cluster state
-        - [ ] Session recording
-        - [ ] Trusted Clusters
-        - [ ] More concepts
-    - [ ] Teleport Auth
-        - [ ] Authentication vs. Authorization
-        - [ ] SSH certificates
-        - [ ] Authentication in Teleport
-        - [ ] Certificate rotation
-        - [ ] Auth API
-        - [ ] Auth state
-        - [ ] Audit log
-        - [ ] Storage back-ends
-        - [ ] More concepts
-    - [ ] Teleport Proxy
-        - [ ] Connecting to a Node
-        - [ ] Recording Proxy mode
-        - [ ] More concepts
-    - [ ] Trusted Clusters
-        - [ ] Introduction
-        - [ ] Join Tokens
-        - [ ] RBAC
-        - [ ] Updating Trusted Cluster role map
-        - [ ] Using Trusted CLusters
-        - [ ] Sharing user traings between Trusted Clusters
-        - [ ] How does it work?
-        - [ ] Troubleshooting
-- [ ] Advanced Features Section
-    - [ ] Enhanced Session Recording
-        - [ ] Requirements
-        - [ ] Step 1/5. Check / Patch kernel
-        - [ ] Step 2/5. Install BCC tools
-        - [ ] Step 3/5. Install and configure Teleport Node
-        - [ ] Step 4/5. Test by logging into node via Teleport
-        - [ ] Step 5/5. Inspect logs
+- [x] Home (Renamed from documentation)
+    - [x] Introduction
+    - [x] Adopters
+    - [x] Getting Started
+    - [x] FAQ
+    - [x] Changelog
+- [ ] Setup (renamed from infrastructure guides)
+    - [x] Installation (moved from Home Section)
+    - [x] Docker Compose
+    - [ ] Cloud Infrastructure
```

Comment:
I'm not sure if we ruled decisively on where these should go ... I vote for here since these will be much briefer.